aws/resource_aws_appstream_test.go

```diff
@@ -0,0 +1,30 @@
+package aws
+
+import (
+	"testing"
+)
+
+func TestAccAWSAppStreamResource_serial(t *testing.T) {
```

Comment:
I'm assuming these cannot be run parallel? Just want to double check because as much as possible, parallel is better.